docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md

```diff
@@ -37,7 +37,7 @@ The page lists the configured and available third-party integrations.
 
 ### Manage integrations
 
-You can add, edit, and delete export configurations for the following tools. You can't delete a configuration that is assigned to a cluster.
+You can add and delete export configurations for the following tools. You can't delete a configuration that is assigned to a cluster.
```

Comment:
why removed edit? Is it because you have to still do those 4 steps steps that involves creating a new one rather than actually editing an existing one?

Reply:
yup